File: tickers/src/main/java/br/com/jeffersonbraga/tickers/model/ticker/Ticker.java
package br.com.jeffersonbraga.tickers.model.ticker;

import javax.persistence.*;
import java.io.Serializable;

@Entity
@Table(name = "TICKER")
public class Ticker implements Serializable {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID")
    private Long id;

    @Column(name = "TICKER")
    private String ticker;

    public Ticker() {

    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getTicker() {
        return ticker;
    }

    public void setTicker(String ticker) {
        this.ticker = ticker;
    }
}

File: tickers/src/main/java/br/com/jeffersonbraga/tickers/model/ticker/Ticker.kt
package br.com.jeffersonbraga.tickers.model.ticker

import java.io.Serializable
import javax.persistence.*

@Entity
@Table(name = "TICKER")
class Ticker : Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "ID")
    var id: Long? = null

    @Column(name = "TICKER")
    var ticker: String? = null
}